config/config.go

```diff
@@ -72,6 +72,10 @@ type MenderShellConfigFromFile struct {
 	Sessions SessionsConfig `json:"Sessions"`
 	// Reconnect interval
 	ReconnectIntervalSeconds int
+	// Disable the Remote Terminal feature
+	DisableRemoteTerminal bool
+	// Disable the port forward feature
+	DisablePortForward bool
```

Comment:
Double negatives. Would it be more natural to call them `EnablePortForward` and `EnableRemoteTerminal`? This also plays a bit better with the opt-in strategy we want to use for these features.

Reply:
This is what we decided to implement for all the other features: opt-out.
See: https://github.com/mendersoftware/mender-connect/blob/master/config/config.go#L39